src/sentry/models/organizationmember.py

```diff
@@ -235,19 +235,25 @@ def get_avatar_type(self):
 
     def get_audit_log_data(self):
         from sentry.models import Team
+        teams = Team.objects.filter(
+            id__in=OrganizationMemberTeam.objects.filter(
+                organizationmember=self,
+                is_active=True,
+            ).values_list('team', flat=True)
+        )
+
         return {
             'email':
             self.email,
             'user':
             self.user_id,
             'teams':
             list(
-                Team.objects.filter(
-                    id__in=OrganizationMemberTeam.objects.filter(
-                        organizationmember=self,
-                        is_active=True,
-                    ).values_list('team', flat=True)
-                ).values_list('id', flat=True)
+                teams.values_list('id', flat=True)
+            ),
+            'team_slugs':
+            list(
+                teams.values_list('slug', flat=True)
```

Comment:
doing this will result in two queries. i'd do something like:
```python
teams = list(Team.objects.filter(
    id__in=OrganizationMemberTeam.objects.filter(
        organizationmember=self,
        is_active=True,
    ).values_list('team', flat=True)
).values('id', 'slug'))
```
and then something like `'teams':  [t['id'] for t in teams]` and then the same for slugs

Reply:
^ good catch

(aside you're always safer to just look up data later rather than cache it, as if it becomes out of date its going to be confusing to show to the end-user)